fullstack/ruby/price_calculator/lib/cart/managers/cart_manager.rb

```diff
@@ -0,0 +1,10 @@
+class CartManager
+  def initialize(base_price_manager:, json_file:)
+    @base_price_manager = base_price_manager
+    @data = JSON.parse(File.read(json_file)).map { |cart_item| CartItem.from_hash(cart_item) }
+  end
+
+  def calculate_total_prices
+    raise NotImplementedError
+  end
+end
```

Comment:
```suggestion
end

```

EOL is inconsistent.